Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/TaskFactory.kt

```diff
@@ -0,0 +1,20 @@
+package de.rki.coronawarnapp.task
+
+interface TaskFactory<
+    ProgressType : Task.Progress,
+    ResultType : Task.Result
+    > {
+
+    interface Config {
+        val collisionBehavior: CollisionBehavior
+
+        enum class CollisionBehavior {
+            ENQUEUE,
+            SKIP_IF_ALREADY_RUNNING
```

Comment:
I know I can be a PITA if it comes to naming... I started thinking about what would be the difference between `SKIP_IF_ALREADY_RUNNING` and `SKIP_IF_STILL_RUNNING`. IMO, it depends on the perspective and on the understanding that this Config is for tasks of the same type.
To be very explicit, how about `SKIP_IF_SIBLING_RUNNING`? That's pretty much exactly describing what the TaskController is doing.